### PR TITLE
Page D'accueil

### PR DESCRIPTION
**Pull request by MoubarkiS**

### CODE REVIEW

**Review by Fatimata-dev — APPROVED** [review]

File: src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+    <img class="mobile-image" src="assets/pictures/LevelUpTasks.jpg" alt="Mobile Header Image">
+  </div>
+</section>
+<section class="description">
+  <p>Bienvenue sur LevelUpTasks, votre solution complète pour la gestion des tâches. Organisez vos projets, créez des expéditions, et collaborez efficacement avec votre équipe. Simplifiez votre workflow et augmentez votre productivité avec notre interface intuitive et nos outils puissants.</p>
+</section>
+<section>
+  <nav>
+    <ul>
+      <button><a href="#">Inscription</a></button>
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: wrap your buttons into li tags

File: src/app/pages/home/home.component.html [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+      <button><a href="#">Connexion</a></button>
+    </ul>
+    <h1>Les Fonctionnalités</h1>
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: only one h1 in this page, use h2 instead

Comment (Benja-san) [reply src/app/pages/home/home.component.html]: Also this title has its place into the below section, not into the buttons nav

File: src/app/pages/home/home.component.html [hunk 3]
```diff
@@ -1,2 +1,60 @@
...
+  </nav>
+</section>
+<section>
+  <div class="content">
+    
+    <div class="image-container">
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: ul and li since you'll have a list of 2 cards

File: src/app/pages/home/home.component.html [hunk 4]
```diff
@@ -1,2 +1,60 @@
...
+      <div class="image-item">
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: use article tag instead of div

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/home/home.component.html [hunk 5]
```diff
@@ -1,28 +1,32 @@
...
+      
     </ul>
-    <h1>Les Fonctionnalités</h1>
+    
   </nav>
 </section>
 <section>
-  <div class="content">
+  <h2>Les Fonctionnalités</h2>
+  <article class="content">
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: no article tag here

File: src/app/pages/home/home.component.html [hunk 6]
```diff
@@ -1,28 +1,32 @@
...
     
-    <div class="image-container">
-      <div class="image-item">
+    <ul class="image-container">
+      <li class="image-item">
         
         <h3>Créer des taches !</h3>
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: wrap all your card content into article tag

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/app.routes.ts [hunk 7]
```diff
@@ -25,11 +29,11 @@ export const routes: Routes = [
     component: TaskPageComponent,
   },
   {
-    path: 'inscription',
-    component: InscriptionComponent,
+    path: 'signup',
```
Comment (Benja-san) [inline src/app/app.routes.ts]: code in english but content in french ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**